proto/ociregistry.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package ociregistry;
+
+message OCIActionResultKey {
```

Comment:
Please add some documentation to the proto.

Reply:
Added! Please note any unclear spots or questions you still have after reading the docs.